src/plugins/data_views/docs/openapi/paths/api@data_views.yaml

```diff
@@ -33,5 +33,11 @@ get:
           examples:
             getAllDataViewsResponse:
               $ref: '../components/examples/get_data_views_response.yaml'
+    '400':
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            $ref: '../components/schemas/400_response.yaml'
```

Comment:
@mattkime OpenAPI requires a 4xx response. The [current docs](https://www.elastic.co/guide/en/kibana/current/data-views-api-get-all.html) for this API don't include one. Let me know if this generic 400 example is okay.